jmx-metrics/src/main/groovy/io/opentelemetry/contrib/jmxmetrics/InstrumentHelper.groovy

```diff
@@ -130,8 +135,11 @@ class InstrumentHelper {
         }
     }
 
-    private static Map<String, String> getLabels(GroovyMBean mbean, Map<String, Closure> labelFuncs) {
+    private static Map<String, String> getLabels(GroovyMBean mbean, Map<String, Closure> labelFuncs, Map<String, Closure> additionalLabels) {
         def labels = [:]
+        additionalLabels.each {label, labelFunc ->
```

Comment:
I wonder if these should take priority over standard label functions since they are on an attribute basis.

Reply:
Makes sense to me